en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md

```diff
@@ -1578,6 +1685,39 @@ Follow the instructions below to move all the existing API Manager configuration
         TIME_CREATED TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
         PRIMARY KEY (UUID)
         )
+        /
+
+                CREATE TABLE AM_API_CATEGORIES (
```

Comment:
Formatting issue

Reply:
Fixed 807962bb22325f5a0317272fd821f085cc62a385
Analytics scritps have been added to other versions as well